### PR TITLE
gui-wm/hikari: new wayland window manager

### PR DESCRIPTION
**Pull request by epsilon-0**
CWM inspired window manager based on wlroots

Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

### CODE REVIEW

File: dev-libs/libucl/libucl-0.8.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+IUSE="lua +regex signatures static-libs urlfetch utils"
+DEPEND="!!dev-libs/ucl
+	lua? ( >=dev-lang/lua-5.1:= )
+	signatures? ( dev-libs/openssl:0 )
+	urlfetch? ( net-misc/curl )"
+RDEPEND="${DEPEND}"
+
+DOCS=( README.md doc/api.md )
+
+src_prepare() {
```
Comment (thesamesam) [inline dev-libs/libucl/libucl-0.8.1.ebuild:31]: Drop eapply_user, call default instead.

File: dev-libs/libucl/libucl-0.7.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (thesamesam) [inline dev-libs/libucl/libucl-0.7.3.ebuild:4]: Do you need to revive this old version w/ EAPI 6?

File: dev-libs/libucl/libucl-0.8.1.ebuild (hunk 1 above)
Comment (epsilon-0) [reply dev-libs/libucl/libucl-0.8.1.ebuild:31]: done

File: dev-libs/libucl/libucl-0.7.3.ebuild (hunk 2 above)
Comment (epsilon-0) [reply dev-libs/libucl/libucl-0.7.3.ebuild]: done

File: dev-libs/libucl/libucl-0.8.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	default
+	eautoreconf
+}
+
+src_configure() {
+	local myeconf=""
+	use urlfetch && myeconf="--with-urls"
```
Comment (thesamesam) [inline dev-libs/libucl/libucl-0.8.1.ebuild:38]: do usex instead in econf? or use_with

**Comment by jonasstein** [comment]
Please add 
`Closes: https://bugs.gentoo.org/719672`
see: https://www.gentoo.org/glep/glep-0066.html#commit-messages

**Comment by epsilon-0** [comment]
> Please add
> `Closes: https://bugs.gentoo.org/719672`
> see: https://www.gentoo.org/glep/glep-0066.html#commit-messages

lol, I opened and forgot to add, damn. Must've been really sleepy.

Thanks a tonne, @jonasstein

File: dev-libs/libucl/libucl-0.8.1.ebuild (hunk 3 above)
Comment (epsilon-0) [reply dev-libs/libucl/libucl-0.8.1.ebuild]: done

**Comment by epsilon-0** [comment]
@flappyports another one from my side, no rush. thanks a lot!

**Comment by epsilon-0** [comment]
@flappyports small reminder ping

**Pull request closed without merging**